packages/ilib-lint-common/README.md

```diff
@@ -19,7 +19,7 @@ the ilib-lint tool.
 
 ## Full API Docs
 
-See the [full API docs](https://github.com/ilib-js/ilib-lint-common/docs/ilib-lint-common.md) for more information.
+See the [full API docs](https://github.com/iLib-js/ilib-mono/tree/main/packages/ilib-lint-common/tree/main/docs) for more information.
```

Comment:
I get this when I click on it:

```
404 - page not found
The main branch of ilib-mono does not contain the path packages/ilib-lint-common/tree/main/docs
```

The new link is missing the final part (ilib-lint-common.md)

Reply:
```suggestion
See the [full API docs](https://github.com/iLib-js/ilib-mono/tree/main/packages/ilib-lint-common/tree/main/docs/ilib-lint-common.md) for more information.
```

Or alternately, we could modify the package.json for all packages to output "index.md" from the "doc" script and always point at that?

Reply:
Fixed in 9ad26db.

We can look into unifying documentation across packages later.